src/app/components/Curation/CurationGrid/index.stories.tsx

```diff
@@ -19,4 +21,15 @@ export default {
   parameters: { chromatic: { disable: true } },
 };
 
-export const Example = Component;
+export const Example = () => {
+  return (
+    <Component summaries={fixture.data?.curations[0].summaries} />
+  );
+};
+
+
+export const HighImpactPromo = () => {
+  return (
+    <Component summaries={highImpactPromoFixture.data.curations[0].summaries as Summary[]} />
+  );
+};
```

Comment:
Do we need this example here? Think it's okay to remove as we have it within it's own story under the `HighImpactPromo` directory

Reply:
Makes sense - happy to leave it as is !